src/Data/RouteData.ts

```diff
@@ -8,77 +8,83 @@ export class RouteData {
             'name': 'Puzztool',
             'categories': [
                 {
-                    'name': 'Ciphers - Scrambled Letters',
+                    'name': 'Ciphers',
+                    'description': 'Scrambled letters',
+                    'rootUrl': '/cipher',
                     'children': [
                         {
                             'name': 'Caesar',
-                            'url': '/cipher/caesar',
+                            'url': '/caesar',
                             'description': 'Letters are shifted by a fixed value'
                         },
                         {
                             'name': 'Vignere',
-                            'url': '/cipher/vigenere',
+                            'url': '/vigenere',
                             'description': 'Letters are mapped based on a key word'
                         },
                         {
                             'name': 'Caesar',
-                            'url': '/cipher/autokey',
+                            'url': '/autokey',
                             'description': ' A Vignere cipher which uses plaintext as the key'
                         }
                     ]
                 },
                 {
-                    'name': 'Encoding - Different Ways To Represent Letters',
+                    'name': 'Encodings',
+                    'description': 'Different Ways To Represent Letters',

```

Comment:
Casing is inconsistent - The description of Ciphers only capitalizes the first word, while this does a lot more.  I prefer Sentence casing (only capitalizing the first one), but please standardize on one.